### PR TITLE
Docs: More Engaging Readme

### PR DESCRIPTION
**Pull request by codeallthethingz**
This PR makes the root README.md a little more attractive.  Notably, the top part now has the logo, and the resources section has some images.  I'll take some screenshots as the images won't render until the PR is merged.

I also added a build badge @tristanls 

image
image
image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,26 @@
...
 
 # Getting Started
 
-You can find detailed instructions on how to install the project requirements and how
-to get started [here](https://thousandbrainsproject.readme.io/docs/getting-started)
+You can find detailed instructions on how to install the project requirements and how to get started [here](https://thousandbrainsproject.readme.io/docs/getting-started)
 
 # Current Performance
-We regularly evaluate this system against a set of sensorimotor tasks, summarized in the **[benchmark experiments](./benchmarks/configs/)**. Any time a functional change is made to the code, these experiments are rerun, and results are updated.
+We regularly evaluate this system against a set of sensorimotor tasks, summarized in the [benchmark experiments](./benchmarks/configs/). Any time a functional change is made to the code, these experiments are rerun, and results are updated.
```
Comment (vkakerbeck) [inline README.md:21]: What's the reason you unbolded this?

**Review by vkakerbeck — APPROVED** [review]
Beautiful ✨ Great idea adding some more images, it makes it look much nicer :)

**Comment by vkakerbeck** [comment]
Oh one other thing that would be nice to add is a link to monty_lab for some example projects and experiments our team is working on.

File: README.md (hunk 1 above)
Comment (codeallthethingz) [reply README.md:21]: It seemed inconsistent.  But then I added a bolded link in my changes.  🤷 :)  I'll revert.

**Comment by codeallthethingz** [comment]
@vkakerbeck I'll add the monty labs to the readme in a separate PR.

File: README.md (hunk 1 above)
Comment (vkakerbeck) [reply README.md]: okay yeah I don't have a strong opinion on this. You can have it however you think it looks better in the rendered version. Was just curious if there was something that didn't work with bolding a link.

**Comment by tristanls** [comment]
Looks great!

**Pull request merged**